src/test/java/org/wmn4j/notation/elements/KeyTest.java

```diff
@@ -0,0 +1,54 @@
+
+package org.wmn4j.notation.elements;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Matias Wargelin
+ */
+public class KeyTest {
+	
+	public KeyTest() {
```

Comment:
If the empty constructor and the setUpClass and tearDownClass methods are not needed, they could be removed.

Reply:
Done :+1: